lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -88,7 +88,10 @@ object EstimationHandler extends CohortHandler {
 
     lazy val earliestStartDateForAMonthlySub =
       for {
-        randomFactor <- Random.nextIntBetween(0, 3)
+        yearAgo <- Clock.localDateTime.map(_.toLocalDate.minusYears(1))
+        randomFactor <-
+          if (subscription.termStartDate.isBefore(yearAgo)) Random.nextIntBetween(0, 3)
```

Comment:
Is `termStartDate` the right value to check here, I'd've thought it was the `contractAcceptanceDate`? cc @kelvin-chappell

Reply:
Yes that would be better